cmd/initialize/wizard/survey_answers.go

```
@@ -15,6 +15,7 @@ type surveyAnswers struct {
 }
 
 func (s surveyAnswers) updateManifest(m *manifest.Manifest) {
+	// TODO: handle error here?
```

Unsure what this comment refers to?

---

the next line, but i will move it to the end of the line. currently the error is ignored there and I dont think it should be

---

but its not important to this PR